modules/ROOT/pages/ps-create-configure.adoc

```diff
@@ -0,0 +1,368 @@
+= Create and Configure Private Spaces
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+
+To extend your network to Anypoint Platform, create a _private space_, 
+a virtual, private, and isolated network hosted in CloudHub 2.0,
+to deploy your apps to.
+
+
+== Create a Private Space 
+
+//SELECT PRIVATE SPACE SHARED
+include::partial$select-private-space.adoc[tag=selectPrivateSpace]
+. Click *Create private space*.
+. Enter a xref:ps-gather-setup-info.adoc#name[name] for your private space and click *Create*.
+
+
+== Create a Private Network
+
+[IMPORTANT]
+You can't change the private network configuration after it's created.
+To change the configuration, delete the private network and create it again.
```

Comment:
"delete the private space" - we've removed the ability to delete just private networks in the space.

Reply:
Updated